db/schema.rb

```diff
@@ -425,6 +498,19 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "friendly_id_slugs", force: true do |t|
```

Comment:
All of the other new table definitions are name-spacing themselves to the cost_calculator_builder, but not this one, any reason why?